sunpy/visualization/animator/wcs.py

```diff
@@ -110,7 +103,10 @@ def _get_wcs_labels(self):
         Read first the axes names property of the wcs and fall back to physical types.
         """
         # world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.
-        world_axis_names = getattr(self.wcs, "world_axis_names", [''] * self.wcs.world_n_dim)
+        try:
+            world_axis_names = self.wcs.world_axis_names
+        except AttributeError:
+            world_axis_names = [''] * self.wcs.world_n_dim
```

Comment:
```suggestion
        world_axis_names = self.wcs.world_axis_names
```
I think this is all we need.

Reply:
I thought we still needed to generate the list of strings but [we don't](https://github.com/astropy/astropy/blob/master/astropy/wcs/wcsapi/low_level_api.py#L301) as that's the default in the base class. So @nabobalis's suggestion is correct.